src/google_cloud_sdk_emulators/mod.rs

```diff
@@ -33,23 +33,35 @@ pub const PUBSUB_PORT: u16 = 8085;
 /// Can be rebound externally via [`testcontainers::core::ImageExt::with_mapped_port`]
 ///
 /// [`Spanner`]: https://cloud.google.com/spanner
-pub const SPANNER_PORT: u16 = 9010;
+#[deprecated(since = "0.13.0", note = "please use `SPANNER_GRPC_PORT` instead")]
+pub const SPANNER_PORT: u16 = SPANNER_GRPC_PORT;
+/// Port that the [`Spanner`] emulator container has internally (gRPC)
+/// Can be rebound externally via [`testcontainers::core::ImageExt::with_mapped_port`]
+///
+/// [`Spanner`]: https://cloud.google.com/spanner
+pub const SPANNER_GRPC_PORT: u16 = 9010;
+/// Port that the [`Spanner`] emulator container has internally (REST)
+/// Can be rebound externally via [`testcontainers::core::ImageExt::with_mapped_port`]
+///
+/// [`Spanner`]: https://cloud.google.com/spanner
+pub const SPANNER_REST_PORT: u16 = 9020;
 
 #[allow(missing_docs)]
 // not having docs here is currently allowed to address the missing docs problem one place at a time. Helping us by documenting just one of these places helps other devs tremendously
 #[derive(Debug, Clone)]
-pub struct CloudSdkCmd {
-    pub host: String,
-    pub port: u16,
-    pub emulator: Emulator,
+struct CloudSdkCmd {
+    host: String,
+    port: u16,
+    emulator: Emulator,
+    additional_args: Vec<String>,
 }
 
 #[allow(missing_docs)]
 // not having docs here is currently allowed to address the missing docs problem one place at a time. Helping us by documenting just one of these places helps other devs tremendously
```

Comment:
If it is not pub, we don't need to be super-specific here

```suggestion
/// The emulator used by the sdk
```